patient-appointment-management/.env

```diff
@@ -0,0 +1,88 @@
+# description: Customer name to be displayed in SMS.
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_NAME=
+
+# description: Customer short name. Must be all lowercase one word without any spaces.
+# format: text
+# required: true
+# configurable: true
+APPLICATION_CUSTOMER_CODE=
+
+# description: Customer EHR endpoint url for communication from Twilio via HTTP POST
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_EHR_ENDPOINT_URL=
+
+# description: Twilio phone number to send SMS from. Your patient will see this number.
+# format: phone_number
+# required: true
+# configurable: true
+TWILIO_PHONE_NUMBER=
+
+# description: Reminder outreach window start time local timezone (HHMM) inclusive. default 0000
+# format: text
+# required: true
+# configurable: true
+# default: 0000
+REMINDER_OUTREACH_START=
+
+# description: Reminder outreach window end time local timezone (HHMM) exclusive, default 2400
+# format: text
+# required: true
+# configurable: true
+# default: 2400
+REMINDER_OUTREACH_FINISH=
+
+# description: Reminder 1st outreach offset (HHMM). default 4800
+# format: text
+# required: true
+# configurable: true
+# default: 4800
+REMINDER_FIRST_OFFSET=
+
+# description: Reminder 2nd outreach offset (HHMM). default 2400
+# format: text
+# required: true
+# configurable: true
+# default: 2400
+REMINDER_SECOND_OFFSET=
+
+# description: Twilio studio flow SID for this application
+# format: sid
+# configurable: false
+TWILIO_FLOW_SID=
+
+# description: AWS access key id to your AWS account with admin-level privilege
```

Comment:
Consider a link to the AWS docs describing how to get this.

Reply:
I removed this as the application finds it automatically.